contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py

```diff
@@ -239,11 +245,57 @@ def _transmit_without_retry(self, envelopes):
                 text,
             )
 
+    def enable_standard_metrics(self):
+        # Sets up stats related objects to begin
+        # recording standard metrics
+        stats_ = stats.stats
+        view_manager = stats_.view_manager
+        stats_recorder = stats_.stats_recorder
+
+        # Standard metrics uses a separate instance of MeasurementMap
+        # from regular metrics but still shares the same underlying
+        # map data in the context. This way, the processing during
+        # record is handled through a separate data structure but
+        # the storage only uses one single data structure
+        # Uniqueness should be based off the name of the view and
+        # measure. A warning message in the logs will be shown
+        # if there are duplicate names
+        self.standard_metrics_map = stats_recorder.new_measurement_map()
```

Comment:
Consider decoupling the actual logic from the exporter class. We will need to add several default metrics, need to control the size of this file and have a better organized way.

Reply:
Yes I will do this.